### PR TITLE
15.0 fix payment bitcoin checkout

### PR DESCRIPTION
**Pull request by alihashaam**
https://git.initos.com/nitrokey/odoo-modules/-/issues/1204

### CODE REVIEW

File: payment_bitcoin/models/bitcoin.py [hunk 1]
```diff
@@ -182,11 +182,7 @@ def cron_bitcoin_payment_reconciliation(self):
             [("provider", "=", "bitcoin")]
         )
         payment_journal_obj = acquirer_obj.journal_id
-        check_hours = (
-            self.env["ir.config_parameter"]
-            .sudo()
-            .get_param("payment_bitcoin.bit_coin_order_older_than", "6")
```
Comment (jans23) [inline payment_bitcoin/models/bitcoin.py:188]: It should be `bitcoin_order_older_than` instead of `bit_coin_order_older_than`.

Comment (jans23) [reply payment_bitcoin/models/bitcoin.py:188]: I see it doesn't originate from you but still..

Comment (alihashaam) [reply payment_bitcoin/models/bitcoin.py:188]: it is fixed now

Comment (jans23) [reply payment_bitcoin/models/bitcoin.py:188]: CI fails

**Pull request merged**